libs/net/kafka/KafkaListener`.cs

```diff
@@ -283,7 +283,7 @@ public void Resume()
     /// <param name="partitions"></param>
     public void Resume(IEnumerable<TopicPartition> partitions)
     {
-        if (this.IsPaused)
+        if (this.IsPaused || !this.IsConsuming)
```

Comment:
Without this Kafka never appears to get going again after a failure in the service.